### DISCUSSION TITLE
Find the number such that ... 🧮

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1152] [Galaxy Trivia] [answered]
... its double is 1️⃣4️⃣ more than its quarter?

**Comment by tomerf** [comment, marked as answer]
8, it's double is 16 and it's quarter is 2, so 16-2=14.

**Reply by ParanoidUser** [reply]
Congratulations! 👏👏 You have just earned the prestigious [Silver 🥈 Galaxy Brain](https://github.com/tomerf?achievement=galaxy-brain&tab=achievements) badge. Keep up the great work and unlock the Golden 🥇 badge next by answering 32 questions. Keep it up! 😊🎉

**Comment by CelioAmaral** [comment]
Let the number be x.

ATP,

=> 2x-x/4 = 14

=> 8x-x/4 = 14

=> 7x = 56

=> x = 8

The number is 8.

Lets verify:

2*8 = 16

8/4 = 2

16 exceeds 2 by 14.

This proves that our answer is correct.
=> The number is 8.

**Reply by ParanoidUser** [reply]
Not exactly sure how `2x-x/4 = 14` turned into `8x-x/4 = 14`.

**Reply by CelioAmaral** [reply]
2x - x/4 = 14 
=> Least common multiple(MMC) = 4

8x - x = 56
7x = 56
x=8

To disregard 8x-x/4 = 14 ; D
